README.md

```diff
@@ -7,8 +7,48 @@ The AmpliPi design is entirely open source, from the software, to the firmware,
 
 The system is self-hosted on the Raspberry Pi and is privacy-centric. By design, AmpiPi doesn’t listen to you or spy on you — it just plays your audio! The way it should be. An internet connection is only needed for some external streaming sources, such as Pandora. The Python application running on the Pi hosts a mobile-friendly website and REST API used to control the system. The website is built on top of the REST API.
 
+![High-level Schematic](docs/imgs/Signal_Diagram_with_Expansion_Out.png)
```

Comment:
"Sources" are used twice in this diagram, once for digital/analog sources and once for the 4 analog multiplexer sources. Based on the rest of our naming conventions, the digital/analog sources should be renamed to "streams".

Reply:
I am going to leave this as-is for now, this nomenclature is something that I would like to revisit when we make changes to how sources work.